### ISSUE TITLE
[COGNITION]: GIBCT - Know before you go video has a logo with nondescript alt text

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#3920]
## Description

The VA 508 office has identified the Know Before You Go embedded video has a nondescript `alt` tag for the VA logo. This could be confusing to assistive technology users and should be updated to an accurate description. Screenshot attached below.

I've assigned this to triage, because I'm not sure individual teams can get at the embed code required to make this fix. If the code is readily available, please re-assign to the GIBCT team.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a screen reader user, I want to hear an accurate image description read aloud when I arrow to the VA logo in the Know Before You Go video

## WCAG or Vendor Guidance (optional)

* [Non-text Content: Understanding SC 1.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html)

## Screenshots or Trace Logs

![Screen Shot 2019-12-03 at 10 48 20 PM](https://user-images.githubusercontent.com/934879/70115090-41651a00-1624-11ea-8c9c-7cec904110fa.png)

**Comment by alexpappasoddball** [comment]
@mattpointzxer0 This one is coming your way by triage!

**Comment by joshkimux** [comment]
@artsymartha68 for awareness
